src/index.ts

```diff
@@ -1,46 +1,48 @@
-import { DocumentNode } from "graphql";
-import { TypedDocumentNode } from "@graphql-typed-document-node/core";
+import type { DocumentNode } from "graphql";
+import type { TypedDocumentNode } from "@graphql-typed-document-node/core";
+
+import type { AnyObject, Maybe } from "./helpers";
 
-export type AnyObject = Record<string, any>;
 export type HttpHeaders = Record<string, string | undefined>;
 
 export interface IDBObject {
   id: string;
   createdAt: Date;
   updatedAt: Date;
   deletedAt: number;
-  createdById: string | null;
+  createdById: Maybe<string>;
 }
 
-export type GqlRequest = <
-  ResultT = AnyObject,
-  VariablesT = AnyObject
->(
+export type GqlRequest = <ResultT = AnyObject, VariablesT = AnyObject>(
   query: string | DocumentNode | TypedDocumentNode<ResultT, VariablesT>,
   variables?: VariablesT,
   options?: {
     checkPermissions?: boolean;
-    headers?: HttpHeaders
+    headers?: HttpHeaders;
   }
 ) => Promise<ResultT>;
 
+export type InvokeFunctionCallback = <
+  ResponseT extends InvokeFunctionResponse = InvokeFunctionResponse,
+  ArgsT = AnyObject
+>(
+  name: string,
+  args?: ArgsT,
+  options?: { waitForResponse: boolean; checkPermissions?: boolean }
+) => Promise<ResponseT>;
+
 export type FunctionContext = {
   api: {
     gqlRequest: GqlRequest;
+    /** @deprecated Method is not implemented */
```

Comment:
I think better to remove it completely, actually it's not deprecated as you see from message.